scripts/auto-rebase/rebase.py

```diff
@@ -56,33 +67,40 @@ def try_get_env(var_name, die=True):
 
 
 def run_rebase_sh(release_amd64, release_arm64, release_lvms):
+    """Run the 'rebase.sh' script with the given release versions and return the script's output."""
     script_dir = os.path.abspath(os.path.dirname(__file__))
     args = [f"{script_dir}/rebase.sh", "to", release_amd64, release_arm64, release_lvms]
     logging.info(f"Running: '{' '.join(args)}'")
     start = timer()
-    result = subprocess.run(args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, universal_newlines=True)
+    result = subprocess.run(
+        args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, universal_newlines=True, check=False)
     logging.info(f"Return code: {result.returncode}. Output:\n" +
                  "==================================================\n" +
                  f"{result.stdout}" +
                  "==================================================\n")
     end = timer() - start
-    logging.info(f"Script returned code: {result.returncode}. It ran for {end/60:.0f}m{end%60:.0f}s.")
+    logging.info(f"Script returned code: {result.returncode}. " +
```

Comment:
There are still a lot of `line-too-long` issues, and I think we should turn this rule off. Because it's reducing code readability with not so impactful code changes. WDYT?

Reply:
As discussed, disabled `line-too-long` check